source/door_b_enb.rst

```diff
@@ -0,0 +1,174 @@
+* Ask the Lead Systems Engineer and the Flight Director to send the command 1MCMDBDS to
+  disable the door mechanism, side B. They may wish to test the command link by first
+  sending a no-op command to the SI RCTU (CNOOPSI command). Use ``CAP 1438`` as a guide.
+  The side A command would be 1MCMDADS.
+* Contact the GOT Duty Officer to inform that we need the dump data as soon as possible and to
+  email or call us when the dump file is available.
+* Process the dump data and make sure that there is nothing anomalous in the data *BEFORE*
+  the shutdown. We want to know if a new occurrence looks just like the previous occurrences.
+  If yes, it should appear as if in one frame the door mechanism was enabled.
+* Once analysis of the dump data is complete, convene a telecon at the next reasonable moment
+  with the ACIS team and review the diagnosis. The MIT ACIS team (Peter Ford, Bob Goeke, Mark
+  Bautz, and Bev LaMarr) should also be included in the discussion, either in the telecon or
+  via email. If the diagnosis is consistent with previous anomalies, (TBR... proceed with the
+  recovery). If the diagnosis is not consistent with previous spurious command anomalies,
+  stop and start a more involved analysis with the ACIS team.
+
+...possibly useful things left over from the DEA-A poweroff page I cribbed from... TBR
+
+* Identify whether or not additional comm time is needed and if so ask the OC/LSE to request it.
+* Send an email to ``sot_red_alert@head`` and call a telecon with the FOT, SOT, and FDs to brief
+  them on the diagnosis and the plan to develop a CAP to recover.
+* Prepare a CAP and submit it for review to capreview AT ipa DOT harvard DOT edu, and cc: acisdude.
+  It will also be necessary to call the OC/CC to determine which number should be used for the CAP.
+  This CAP will have the following steps:
+
+  - Send a NOP command, CNOOPSI, and monitor command count increment.
+  - Send command 1MCMDBDS to disable side B of the door mechanism. (Side A: 1MCMDADS).
+  - Verify telemetry has returned to normal.
+
+* Execute the CAP at the next available comm. Reloading the flight software patches can take
+  a half an hour, so ensure that there is enough time in the comm to execute the entire procedure.
+* Write a shift report and distribute to ``sot_shift`` to inform the project that ACIS is restored
+  to its default configuration.
+
+
+Impacts
+-------
+
+* While the door mechanism is enabled, any spurious command to move the door will result
+in undesired hardware action.
+
+Relevant Procedures
+-------------------
+
+
+SOT Procedures
+++++++++++++++
+
+* `Open ACIS Camera Housing Door <http://cxc.cfa.harvard.edu/acis/cmd_seq/open_door.ps>`_ and
+* `Close ACIS Camera Housing Door <http://cxc.cfa.harvard.edu/acis/cmd_seq/close_door.ps>`_ contain useful discussion of the door mechanism.
+
+FOT Procedures
+++++++++++++++
+
+* SAP 61032 (Open ACIS door) and
```

Comment:
Do we not have links for these?

Reply:
Apparently there are no links for these really ancient procedures. I imagine they're "retired".